### PR TITLE
Adding a few potential breaking changes

### PR DESCRIPTION
**Pull request by nicolasnoble**
I'm sure there's many more than just these few.

### CODE REVIEW

**Comment by nicolasnoble** [comment]
Aaah, wrong file, I see.

File: doc/core/pending_api_cleanups.md [hunk 1]
```diff
@@ -15,3 +15,4 @@ number:
   `include/grpc/impl/codegen/grpc_types.h` (commit `af00d8b`)
   (cannot be done until after next grpc release, so that TensorFlow can
   use the same code both internally and externally)
+- require a C++ runtime for all languages.
```
Comment (sanjaypujare) [inline doc/core/pending_api_cleanups.md]: "all languages wrapping core"?

**Comment by vjpai** [comment]
How about never

File: doc/core/pending_api_cleanups.md (hunk 1 above)
Comment (nicolasnoble) [reply doc/core/pending_api_cleanups.md]: Done.

**Pull request merged**